gobblin-modules/google-ingestion/src/main/java/gobblin/ingestion/google/webmaster/GoogleWebMasterSource.java

```diff
@@ -159,14 +167,17 @@
   // =============================================
 
   private final static Splitter splitter = Splitter.on(",").omitEmptyStrings().trimResults();
+  private final static String SOURCE_PROPERTY_COLUMN_NAME = "source";
```

Comment:
What if source schema has "source" column name? Should we make this configurable?